src/clojure/puppetlabs/ssl_utils/core.clj

```diff
@@ -255,6 +255,22 @@
   [cert-or-csr :- CertOrCSR]
   (:dns-name (get-extension-value cert-or-csr subject-alt-name-oid)))
 
+(schema/defn ^:always-validate subject-ip-alt-names :- SSLExtension
```

Comment:
I think this needs to be combined with `subject-dns-alt-names` to create an extension that has both `:dns-name` and `:ip` keys?

Reply:
Yeah, I only added this for testing atm.  Lemme see about making one for both and using that for testing instead.

Reply:
OK.  Done.